### DISCUSSION TITLE
New sonde type MTS01

### DISCUSSION THREAD

**Discussion by 73-de-LZ** [dl9rdz/rdz_ttgo_sonde#330] [Developers corner]
Did somebody work on decoder for the new sonde types MTS01, launched from Istanbul, Turkey? I think they are made in Turkey, also I can ask a friend for recordings, but new beta of Auto_RX has successfully decode it.
![Screenshot_20221021-041550_Chrome](https://user-images.githubusercontent.com/16052106/197320468-5873039a-98d0-4124-8d55-9497e9ec1512.jpg)
![Screenshot_20221021-041520_Chrome](https://user-images.githubusercontent.com/16052106/197320477-0de47c3c-e92d-48c9-a2ee-bda7afe11550.jpg)

**Comment by LukePrior** [comment]
This issue contains further information about how support was added in auto Rx

https://github.com/rs1729/RS/issues/48
